components/delete_category_modal.tsx

```diff
@@ -18,11 +18,13 @@ type Props = {
 };
 
 export default class DeleteCategoryModal extends React.PureComponent<Props> {
-    handleCancel = () => {
+    handleCancel = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```

Comment:
It does, I was using the second type because that's what was suggested by IntelliSense. The second type is used to type the `nativeEvent` field in case you need access to fields only available in the native HTML event. For this case we don't need that so I've removed it.